tools/mason/MasonNew.chpl

```diff
@@ -231,6 +219,13 @@ considered if no input is given.""");
   return (packageName, version, chapelVersion);
 }
 
+proc exitOnEOF(parameter) {
```

Comment:
A bonus of refactoring this out into a separate function is that you can add a doc-string and explain why we need this, since it's probably not clear from the code alone :)

Reply:
Okay, I'll make that change along with [that](https://github.com/chapel-lang/chapel/pull/15748#discussion_r435540692)